runtime/src/bank/tests.rs

```diff
@@ -13633,11 +13633,14 @@ fn test_loader_v3_to_v4_migration() {
         .unwrap()
         .copy_from_slice(&elf);
     let message = Message::new(
-        &[solana_loader_v3_interface::instruction::migrate_program(
-            &programdata_address,
-            &program_keypair.pubkey(),
-            &program_keypair.pubkey(),
-        )],
+        &[
```

Comment:
@Lichtso these instructions seem to have been introduced after SIMD-0170, which set a maximum builtin budget of 3_000.

Before this feature was activated, the ixs would get a budget of 200k. After activation, they only get 3k. This means that these instructions need to be paired with a set compute unit limit to succeed.

Reply:
Are loaders also planned to migrate to bpf? Once they are migrated, they will no longer subject to 3K limit.

Reply:
`ComputeBudgetInstruction::set_compute_unit_limit` in the tests is fine, the CLI does handle that for the dApp devs.

> Are loaders also planned to migrate to bpf?
loader-v1 and loader-v2 are deprecated, loader-v3 is about to be deprecated and loader-v4 will only be migrated on-chain later, after loader-v3 is deprecated.